crates/core/c8y_api/src/smartrest/operations.rs

```diff
@@ -55,12 +54,18 @@ impl Operations {
         None
     }
 
-    pub fn filter_by_topic(&self, topic_name: &str) -> Vec<(String, Operation)> {
+    pub fn filter_by_topic(
+        &self,
+        topic_name: &str,
+        record: &impl Record,
+    ) -> Vec<(String, Operation)> {
         let mut vec: Vec<(String, Operation)> = Vec::new();
         for op in self.operations.iter() {
             match (op.topic(), op.on_fragment()) {
                 (None, Some(on_fragment)) => vec.push((on_fragment, op.clone())),
-                (Some(topic), Some(on_fragment)) if topic == topic_name => {
+                (Some(ref topic), Some(on_fragment))
+                    if record.inject_values_into_template(topic) == topic_name =>
```

Comment:
It would be better to inject any `${.bridge...}` config when the operations are loaded. The bridge config will not change after the mapper starts.

```suggestion
                    if topic == topic_name =>
```